02_configure_host.sh

```diff
@@ -54,6 +54,16 @@ if [[ ! -z "${MIRROR_IMAGES}" || $(env | grep "_LOCAL_IMAGE=")  || ! -z "${ENABL
     setup_local_registry
 fi
 
+if [[ ! -z "${SQUID_PROXY}" ]]; then
+  generate_squid_conf > squid.conf
+
+  sudo podman run -d --rm \
```

Comment:
If we are listening at 3128, do we really need a root container instead of a rootless one?

Reply:
We don't run any rootless containers in dev-scripts. They work fine for me, but we've had complaints. I'm happy to move them all, but preferably as a separate PR

Reply:
Rootless podman didn't work back in centos7 days, we could probably move to it now that we only support centos/RHEL8 but agree that should probably be a separate PR